docs/architecture/porting-existing-aspnet-apps/identity-differences.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Comparing ASP.NET Identity and ASP.NET Core Identity
+description: 
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Comparing ASP.NET Identity and ASP.NET Core Identity
```

Comment:
This is the most likely place to call out migrating from the Microsoft.Owin libraries to the Asp.Net Core alternatives since those were primarily used for authentication. Unless that's going to get its own chapter?
https://docs.microsoft.com/en-us/aspnet/core/migration/proper-to-2x/?view=aspnetcore-3.1#globalasax-file-replacement
https://devblogs.microsoft.com/aspnet/katana-asp-net-5-and-bridging-the-gap/

Reply:
Added to ch2 TODO